jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/SqlDialect.java

```diff
@@ -56,6 +56,11 @@ public static String getPublicName() {
         NONE, SINGLE, MULTIPLE, AUTO_DETECT
     }
 
+    // Specifies different exception handling strategies
```

Comment:
Replace comment by JavaDoc (check other enums for examples).

Reply:
fixed